docs/posts/ibis-duckdb-geospatial/index.qmd

```diff
@@ -0,0 +1,150 @@
+---
+title: "Ibis Geospatial: Introducing DuckDB `spatial` support"
+author: Naty Clementi
+date: 2023-12-07
+categories:
+    - blog
+    - duckdb
+    - geospatial
+---
+
+
+Ibis now has support for some DuckDB geospatial functions (check the list [here](https://gist.github.com/ncclementi/fbc5564af709e2d7f8882821e3a8649f)), let's take them for a spin.
+
+This post is inspired in the ["DuckDB: Spatial Relationships"](https://geog-414.gishub.org/book/duckdb/07_spatial_relationships.html) lesson from [Dr. Qiusheng Wu](https://geog-414.gishub.org/book/preface/instructor.html)'s course "Spatial Data Management"
+from the Department of Geography & Sustainability at the University of Tennessee, Knoxville.
+
+I'll convert the examples to use the Ibis, and add a little extra at the end.
+
+::: {.callout-note}
+You can check Dr. Qiusheng Wu's full Spatial Data Management course material in its  [website](https://geog-414.gishub.org/index.html), and the classes are also on [youtube](https://www.youtube.com/watch?v=A4TOAdsXsEs&list=PLAxJ4-o7ZoPe9SkgnophygyLjTDBzIEbi),
+:::
+
+## Setup and Data
+
+At the moment, the geospatial features for the DuckDB backend are available on the main branch of Ibis, but they'll be included in the next release.
+
+To download the data that will be using in this post,
+
+```bash
```

Comment:
This won't run bash code.

I think the only way to do that is using the `!` syntax from IPython